src/analyses/variable-sensitivity/value_set_abstract_object.cpp

```diff
@@ -148,8 +148,12 @@ abstract_object_pointert value_set_abstract_objectt::expression_transform(
     collective_operands,
     [&resulting_objects, this, &expr, &environment, &ns](
       const std::vector<abstract_object_pointert> &ops) {
+      auto operands_expr = exprt::operandst { };
```

Comment:
Everything is else in that little lambda is declared `auto`. For me, it's easier to read if everything is `auto` or everything isn't `auto`. Obviously there are times when that's not possible, because the type deduction doesn't do what we want, but within a scope I generally try to go with one or the other.